NPM unable to resolve dependency on `npm install`

When freshly cloning the template, or create an Atom project from it, installation of NPM dependencies fails:
```
astrofront-astro$ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: Astrofront@2.0.0
npm ERR! Found: astro@5.1.5
npm ERR! node_modules/astro
npm ERR!   astro@"5.1.5" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer astro@"^5.3.0" from @astrojs/node@9.1.0
npm ERR! node_modules/@astrojs/node
npm ERR!   @astrojs/node@"^9.0.0" from the root project
npm ERR! 
```

This is about the `astro` package version, as a direct dependency, and as a peer dependecy of `@astrojs/node`.

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "@astrojs/tailwind": "5.1.4",
     "@astrojs/vercel": "^8.0.1",
     "@nanostores/react": "^0.8.4",
-    "astro": "5.1.5",
+    "astro": "^5.3.0",
     "astro-auto-import": "^0.4.4",
     "astro-font": "^0.1.81",
     "date-fns": "^4.1.0",

